snapcraft/plugins/python2.py

```diff
@@ -168,6 +169,11 @@ def build(self):
              '--prefix={}/usr'.format(self.installdir),
              ], cwd=self.builddir)
 
+        # Fix all shebangs to use the in-snap python.
+        common.replace_in_file(self.installdir, re.compile(r''),
+                               re.compile(r'#!.*python'),
```

Comment:
I can't imagine a scenario where we don't want to use `#!/usr/bin/env python` instead of whatever they have. If they _don't_ use installdir in the shebang it'll still be broken, no?